src/components/Search/Search.jsx

```diff
@@ -13,7 +14,11 @@ class Search extends React.Component {
   }
   render() {
     const { data, pros, ...rest } = this.props;
-    return <SearchResult {...rest} players={data || []} pros={pros || []} />;
+    return (<div>
+      <Helmet title={`Search Results for ${this.props.location.query.q}`} />
```

Comment:
String to string file (use a template like %s and util.format to build the string here)